Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -716,7 +716,7 @@ public struct RenderNodeTranslator: SemanticVisitor {
                 // Collect all child topic references.
                 contentCompiler.collectedTopicReferences.append(contentsOf: groups.flatMap(\.references))
                 // Add the final groups to the node.
-                sections.append(contentsOf: groups.map(TaskGroupRenderSection.init(taskGroup:)))
+                sections.append(contentsOf: groups.map { .init(taskGroup: $0, generated: true) })
```

Comment:
Is the shorter `.init` syntax always preferable in Swift? To be honest for someone like me learning the DocC source code the previous version of this line that spelled out `TaskGroupRenderSection.init` was more readable.

Reply:
I'd say it's a matter of personal preference. My view is that it depends on what information I want to convey. For example: do I need the reader to know that this initializes a `TaskGroupRenderSection` value specifically or do I only need the reader to know that this initializes the same type of value as `sections`?

In this case I had changed it because I added a second argument to the initializer so I needed to switch to an explicit closure anyway but as you pointed out the second argument isn't needed to I changed this line back to what it was.